main.pm

```diff
@@ -99,6 +99,10 @@ if (get_var("LIVETEST")) {
     $testapi::username = "root";
     $testapi::password = '';
 }
+elsif (get_var("AUTOYAST") || get_var("SUPPORT_SERVER") ) {
+    $testapi::username = "root";
+    $testapi::password = "root";
+}
```

Comment:
yes, the autoyast tests in tests/autoyast needs root access. I guess there are now some functions to log in as root, so this redefinition may no longer be needed, and the root password can already be specified in autoyast.xml, as nots3cr3t
I just thought this is preferred way to do this, according to the lines above.